README.md

```diff
@@ -216,14 +227,67 @@ contract LANDAuction is Ownable, LANDAuctionStorage {
     */
     function disableToken(address _address) public onlyOwner;
 
+    /**
+    * @dev Get exchange rate
+    * @param _srcToken - IERC20 token
+    * @param _destToken - IERC20 token
+    * @param _srcAmount - uint256 amount to be converted
+    * @return uint256 of the rate
+    */
+    function getRate(
+        IERC20 _srcToken,
+        IERC20 _destToken,
+        uint256 _srcAmount
+    ) public returns (uint256 rate)
+
     /**
     * @dev Convert allowed token to MANA and transfer the change in MANA to the sender
```

Comment:
Instead of transfer the change in MANA should be something like "transfer the change in the original token" as per the last changes.